Open Judge System/Workers/OJS.Workers.Compilers/CPlusPlusZipCompiler.cs

```diff
@@ -38,17 +26,19 @@ public override string RenameInputFile(string inputFile)
         public override string BuildCompilerArguments(string inputFile, string outputFile, string additionalArguments)
         {
             var arguments = new StringBuilder();
+            var compilingDirectory = $"{Path.GetDirectoryName(inputFile)}\\CompileDir";
```

Comment:
If you added a Compilation Directory in the Base Compiler, why not that instead of randomly creating a new one here?